decidim-meetings/app/views/decidim/meetings/admin/registrations/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title">
+      <%= title %>
+      <span class="exports dropdown tiny button button--simple button--title" data-toggle="export-dropdown"><%= t "actions.export", scope: "decidim.admin" %></span>
+      <div class="dropdown-pane" id="export-dropdown" data-dropdown data-auto-focus="true" data-close-on-click="true">
+        <ul class="vertical menu add-features">
+          <% %w{csv json}.each do |format| %>
```

Comment:
Can you move the array to the Controller, or to a constant somewhere else? I don't think it's a concern of the template knowing which exporters are available.

Reply:
Well.. the problem is we had a good abstraction for this but we detected a problem. @josepjaume implemented a helper to render all available exporters for a resource but this exporter is a bit different so I finally implemented my own version of it.

I think this is kinda big for this PR and we should continue iterating the solution provided by @josepjaume including this new behavior.